.env

```diff
@@ -0,0 +1,2 @@
+PORT=3000
+DATABASE=movies-project
```

Comment:
Acuérdate a no subir nunca los archivos .env a gitHub. En ellos ponemos información sensible y no queremos que la vea nadie. Incluye **_.env_** en el archivo **_.gitignore_**

Reply:
Hola Javi. Soy consciente de ello.  En el de basic authentification por ejemplo también lo vas a a ver...  La plantilla indica lo siguiente:  
# Environment Variables should NEVER be published
# We are submitting it for learning purposes 
# .env